brave/browser-laptop
# Supersede cd3dd7829731f6e36eb70ac1ac4397598e615153 by "scroll-touch-edge"

Auditors: @bbondy

Test Plan:
1. Open google search result
2. Click any link and back to the search result
3. Make the window smaller by open dev console (shift + f8)
4. Make sure you can horizontally scroll the page
5. Make sure you can swipe back/forward when you scroll to the edge
